packages/color-area/src/ColorArea.ts

```diff
@@ -65,7 +65,7 @@ export class ColorArea extends SpectrumElement {
     @property({ type: String, attribute: 'label-y' })
     public labelY = 'luminosity';
 
-    @query('.handle')
+    @query('.handle', true)
```

Comment:
The second argument to @query sets the required option. If the element is not found, it will throw.
If .handle, [name="x"], or [name="y"] elements are missing from the DOM at any point, your code will throw, potentially breaking the component.

Reply:
The second argument is setting the `cache` option. This is so that it doesn't have to query the DOM each time we access `this.handle`. Is there a condition you're concerned about where this element won't be in the DOM?